src/client.ts

```diff
@@ -168,6 +173,37 @@ export class PlainClient {
     return unwrapData(res, (q) => nonNullable(q.createIssue.issue));
   }
 
+  /**
+   * Resolve an issue for a customer.
+   */
+  async resolveIssue(
+    input: VariablesOf<typeof ResolveIssueDocument>['input']
```

Comment:
This feels a little 'raw' for an SDK, this could just be `issueId` and we construct the input

Reply:
That is what it is but just as a named argument (notice the 'input' at the end):

This means this is valid:

```tsx
  const foo = await client.resolveIssue({
    issueId: 'it_123'
  })
```

Reply:
Or did you mean: `resolveIssue('asda')` would be nicer?

Reply:
The latter